sessionctx/binloginfo/binloginfo_test.go

```diff
@@ -147,6 +147,30 @@ func (s *testBinlogSuite) TestBinlog(c *C) {
 	}
 	c.Assert(matched, IsTrue)
 
+	tk.MustExec("drop table if exists t1")
```

Comment:
please add test for the partition table, eg:
```sql
create table t2 (a int,b int,primary key(a,b)) partition by hash(a) partitions 4;
create table t3 (a int key) partition by range(a) (
		partition p0 values less than (10),
		partition p1 values less than (20)
	);
```

Reply:
I don't take care of this case, thx!